content/app/development/configuration/startup/_index.nb.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Konfigurer appens oppførsel under oppstart
+linktitle: Startup
+description: Hvordan konfigurere applikasjonens oppførsel under oppstart av app-frontend
+toc: true
+weight: 500
+---
+
+I `applicationmetadata.json` har man mulighet til å styre hva app-frontend skal foreta seg i det brukeren går mot en direktelenke til applikasjonen.
```

Comment:
Ville det vært mer folkelig å si at man har mulighet til å styre hva applikasjonen viser i det brukeren går mot en direktelenke i stedet for å skape dette skille mot frontend og backend?

Reply:
Enig. Fixed.